tools/pythonpkg/src/python_replacement_scan.cpp

```diff
@@ -24,7 +25,13 @@ static void CreateArrowScan(const string &name, py::object entry, TableFunctionR
 	children.push_back(make_uniq<ConstantExpression>(Value::POINTER(CastPointerToValue(stream_factory_produce))));
 	children.push_back(make_uniq<ConstantExpression>(Value::POINTER(CastPointerToValue(stream_factory_get_schema))));
 
-	table_function.function = make_uniq<FunctionExpression>("arrow_scan", std::move(children));
+	if (type == PyArrowObjectType::PyCapsule) {
+		// Disable projection+filter pushdown
+		table_function.function = make_uniq<FunctionExpression>("arrow_scan_dumb", std::move(children));
```

Comment:
Maybe `arrow_no_pushdown`?

Reply:
Hmm I agree, but it's used by duckdb-java, see the [initial PR](https://github.com/duckdb/duckdb/pull/5449) and the [JDBC Client source code](https://github.com/duckdb/duckdb-java/blob/2dca81c771e5af1db4074d9d7c0e7ab6f5a31ae3/src/jni/duckdb_java.cpp#L1235)

I'm hesitant to rename something that's used by third party clients